eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -94,26 +101,37 @@ function DeployStressTests(
         Login -subscription $subscription -clusterGroup $clusterGroup -pushImages:$pushImages
     }
 
-    RunOrExitOnFailure helm repo add stress-test-charts https://stresstestcharts.blob.core.windows.net/helm/
+    $chartRepoName = 'stress-test-charts'
+    if ($LocalAddonsPath) {
+        $absAddonsPath = Resolve-Path $LocalAddonsPath
```

Comment:
Do you think we need a Test-Path here?

Reply:
We don't because of the validate script on the parameter. But I did realize that I need to be running `helm plugin add` via `RunOrExitOnFailure`, so that's updated now.